scripts/g.extension/testsuite/test_addons_download.py

```diff
@@ -0,0 +1,141 @@
+"""
+TEST:      test_addons_download.py
+
+AUTHOR(S): Stefan Blumentrath <stefan dot blumentrath at nina dot no)
+           based on test_addons_modules.py by
+           Vaclav Petras <wenzeslaus gmail com>
```

Comment:
I appreciate giving credit and tracking origin, but perhaps not even needed in this case, esp. after all the modernization you did (same for copyright line).

Maybe mention in the class or module docstring that it is using the presence of the files as a test like the other test does. (You will also create a useful docstring and Pylint will be happier.)

Reply:
Implemented too